### DISCUSSION TITLE
Rule energy#power

### DISCUSSION THREAD

**Discussion by stouph19** [arendst/Tasmota#17516] [General]
Hello,
I am French, translated by google.
I would like to create a rule on a POWR2 sonoff.
I would like when "energy#power>100" it sends me via MQTT "1" and "energy#power<100" sends me "0".
Can you tell me how to do it?
Thanks in advance

**Comment by sfromis** [comment]
You can use the commands `PowerHigh` and `PowerLow` to get Tasmota to publish messages when thresholds are crossed, without rules.

**Reply by stouph19** [reply]
Great, that's exactly what I wanted.
Thanks very much
